style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  font: 16px/1.5 Arial, Helvetica, sans-serif;
+}
+
+* {
+  text-decoration: none;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+#branding a {
+  font-size: 18px;
+  font-weight: bold;
+  color: rgb(75, 75, 216);
+}
+
+.line {
+  width: 18px;
+  height: 2px;
+  background-color: rgb(75, 75, 216);
+  margin: 5px 0;
+  border-radius: 2px;
+}
+
+.container {
+  margin: auto;
+  padding-top: 40px;
+  width: 92%;
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+.showcase {
+  border-bottom-left-radius: 100px;
+  margin-bottom: 10vh;
+  background-image: url(./images/header-shapes-mobile.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
> - [ ] Please, consider that the content of your headline section is aligned with flexbox

- [ ] As required by the previous reviewer, please use flexbox CSS property to align your `.showcase` contents
you can visit this [link](https://www.w3schools.com/css/css3_flexbox.asp) that explain how to use flex

Reply:
@BenMukebo Thank You so much for the review and guidance :-)
I will make the necessary changes and resubmit.